### ISSUE TITLE
The following packages have been kept back (again)

### ISSUE THREAD

**Issue by maurom** [oerdnj/deb.sury.org#1090]
Yesterday, while trying to update PHP 7.2 on Debian Stretch 9.7, apt informed this:

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  libapache2-mod-php7.2 php7.2-cli php7.2-common php7.2-curl php7.2-gd php7.2-intl php7.2-json php7.2-mbstring php7.2-mysql php7.2-opcache php7.2-readline php7.2-soap
  php7.2-xml php7.2-xmlrpc
0 upgraded, 0 newly installed, 0 to remove and 14 not upgraded.
```

A similar problem was reported by user @Julienh on 2017, tracked as #766. At that time, it was caused by a dependency change on package php7.2-zip, going from libzip4 to libzip5 (see [here](https://github.com/oerdnj/deb.sury.org/issues/766#issuecomment-351456632)).

This time its similar, but going backwards (see _Version_ & _Depends_ lines): 

```
# apt show -a php7.2-zip
Package: php7.2-zip
Version: 7.2.15-1+0~20190209065123.16+stretch~1.gbp3ad8c0
Priority: optional
Section: php
Source: php7.2
Maintainer: Debian PHP Maintainers <team+pkg-php@tracker.debian.org>
Installed-Size: 94,2 kB
Provides: php-zip
Depends: php-common (>= 1:35), ucf, php7.2-common (= 7.2.15-1+0~20190209065123.16+stretch~1.gbp3ad8c0),
    libc6 (>= 2.14), libzip4 (>= 1.3.2), zlib1g (>= 1:1.1.4)
Homepage: http://www.php.net/
Download-Size: 20,3 kB
APT-Sources: https://packages.sury.org/php stretch/main amd64 Packages
Description: Zip module for PHP
 This package provides the Zip module(s) for PHP.
 .
 PHP (recursive acronym for PHP: Hypertext Preprocessor) is a widely-used
 open source general-purpose scripting language that is especially suited
 for web development and can be embedded into HTML.

Package: php7.2-zip
Version: 7.2.13-1+0~20181207100540.13+stretch~1.gbpf57305
Status: install ok installed
Priority: optional
Section: php
Source: php7.2
Maintainer: Debian PHP Maintainers <team+pkg-php@tracker.debian.org>
Installed-Size: 94,2 kB
Provides: php-zip
Depends: php-common (>= 1:35), ucf, php7.2-common (= 7.2.13-1+0~20181207100540.13+stretch~1.gbpf57305),
    libc6 (>= 2.14), libzip5 (>= 1.3.2), zlib1g (>= 1:1.1.4)
Homepage: http://www.php.net/
Download-Size: desconocido
APT-Manual-Installed: yes
APT-Sources: /var/lib/dpkg/status
Description: Zip module for PHP
 This package provides the Zip module(s) for PHP.
 .
 PHP (recursive acronym for PHP: Hypertext Preprocessor) is a widely-used
 open source general-purpose scripting language that is especially suited
 for web development and can be embedded into HTML.
```
And so, apt refuses to install/upgrade packages as it requires downgrading libzip5 to libzip4.

That can be easily fixed in my servers, but I would like to hear your feedback on which libzip version is the right one.

Anyway, keep up the good work. I wholeheartedly appreciate your hard work on packaging PHP.

**Comment by oerdnj** [comment]
libzip4 is the correct version, upstream bumped the SOMAME just to remove one useless symbol and I reverted this change in Debian to avoid useless transition.

**Comment by maurom** [comment]
Ok. Thanks!

**Comment by soifs** [comment]
How is this fixed on debian stretch 9.7?
Should we install latest 1.5.1 from testing repository?
since on stable the latest libzip4 is 1.1.2-1.1+b1
